src/components/screens/signMultiSignTransaction/importData/importData.js

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useEffect } from 'react';
+import Box from '../../../toolbox/box';
+import BoxContent from '../../../toolbox/box/content';
+import BoxFooter from '../../../toolbox/box/footer';
+import { PrimaryButton } from '../../../toolbox/buttons';
+import Feedback from '../../../toolbox/feedback/feedback';
+import transactionTypes from '../../../../constants/transactionTypes';
+import ProgressBar from '../progressBar';
+import styles from './styles.css';
+
+// eslint-disable-next-line complexity
+const isInputValid = ({
+  nonce, fee, type, asset,
+  lsTrackingId, senderPublicKey,
+  signatures,
+}) => {
+  const {
+    amount, recipientId, mandatoryKeys, data,
+    optionalKeys, numberOfSignatures,
+  } = asset;
+
+  if (
+    parseInt(nonce, 10) && parseInt(fee, 10)
+    && type === transactionTypes().transfer.code.new && parseInt(amount, 10)
+    && Array.isArray(signatures) && Array.isArray(optionalKeys) && Array.isArray(mandatoryKeys)
+    && typeof recipientId === 'string' && typeof data === 'string'
+    && typeof lsTrackingId === 'string' && typeof senderPublicKey === 'string'
+    && typeof numberOfSignatures === 'number'
```

Comment:
I would go with a list of validators and `Array.reduce`.  It's more readable, extendible and definitely easier tested.